administrator/language/de-DE/com_installer.ini

```diff
@@ -27,7 +38,11 @@ COM_INSTALLER_EXTENSION_PACKAGE_FILE="Erweiterungspaketdatei"
 COM_INSTALLER_EXTENSION_PROTECTED="Geschützte Erweiterung"
 COM_INSTALLER_EXTENSION_PUBLISHED="Erweiterung aktiviert."
 COM_INSTALLER_EXTENSION_UNPUBLISHED="Erweiterung versteckt."
-COM_INSTALLER_FAILED_TO_ENABLE_UPDATES=", Updates konnten <u>nicht</u> aktiviert werden."
+COM_INSTALLER_FIELD_EXTRA_QUERY_DESC="Diese Erweiterung schränkt Aktualisierungen ein. Typischerweise bedeutet dies, dass es sich um eine Erweiterung mit kostenpflichtigen Downloads handelt. Den Download-Schlüssel erhalten Sie vom Entwickler und geben ihn hier ein, um Updates für die Erweiterung freizuschalten. Möglicherweise muss die Erweiterung  bezahlt werden, um den Download-Schlüssel zu erhalten oder zu aktivieren."
+COM_INSTALLER_FIELD_EXTRA_QUERY_LABEL="Download-Schlüssel"
+COM_INSTALLER_FIELD_NAME_LABEL="Name"
+COM_INSTALLER_FIELD_TYPE_LABEL="Typ"
+COM_INSTALLER_FIELD_TYPE_LOCATION="Ort"
```

Comment:
Ort? Hört sich falsch an wo wird das angezeigt also in welchem Context?

Reply:
sollten wir prüfen.

Reply:
fixed with https://github.com/joomlagerman/joomla/pull/1076/commits/c8ff5b2a6b58be966ffd60d4ae70c45398218895